README.md

```diff
@@ -1 +1,56 @@
-NL-Design-System
+# NL-Design-System
+
+The NL Design Skeleton application is designed for rapid application testing and prototype development on the NL Design System. It provides a basic skeleton application with full NL Design system functionality that any developer can easily extend, view locally and deploy to an online environment for demonstration purposes. The main benefits are:
+
+- Development and (online) demonstration of prototypes without the need of a server.
+
+- An out-of-the-box basic application that doesn't require configuration or setup and can be extended immediately.
+
+This NL-Design-system repository is a customizable and straightforward component library to help make Progressive Web Applications faster, attractive, and more accessible with React.
```

Comment:
Dit klopt beter. Dat het een React component library is, mag denk ik prominenter zijn.